i18n/es.yaml

```diff
@@ -1,5 +1,5 @@
 toggleMenu:
-    other: Toggle Menu
+    other: Esconder Menu
```

Comment:
As we are modifying this translation, we can change it to the correct spelling `menú

Reply:
Changes already made